xAPI.md

```diff
@@ -509,11 +509,47 @@ __Verb__: Defines the action being done by the Actor within the Activity within
+increase flexibility.  The intention in only supplying a basic amount of syntax rules is that Communities of Practice (CoP) can define content (verbs, activity types, contextual relationships, extensions, etc.) for their use cases.  It is very 
+important that such communities exist and share best practices.
+
 ### 8.1 Extending xAPI
 
-### 8.2 Profiles
+xAPI can be extended in a few ways.  The most notable and extensive is Statement Extensions, which allow great flexibility 
+within Statements.  It is recommended that profiles or Communities or Practice agree on how to use extensions for their 
+particular use cases. Implementation details are covered in a <a href="#miscext">later section</a>.  
+
+The About Resource is another way xAPI enables extensions.  The LRS may find it useful to communicate features or 
+behaviors beyond this specification to activity provider.  It is hoped that if more information is available about an 
+LRS, that the About Resource captures it.
+
+Finally, the set of Resources implemented is not expected to be constrained by this document. While not explicitly 
+defined, this document suggests that no new Endpoints defined in future iterations of this specification will 
+begin with /extensions and that LRSs create other custom xAPI Resources there. 
+
+<a name="COPs" />
+
+### 8.2 Profiles and Communities of Practice
+
+A profile of xAPI is a set of rules and/or vocabularies applied to a valid xAPI implementation.  Often, a CoP leverages many best practices to create a profile.
+
+A profile will typically define some of the open-ended areas of the specification, such as identifying verbs, activity types, structure and domains for IRIs, and names/possible values within xAPI Resources like the State API.  This is 
+referred to as creating a vocabulary.
+
+It is recommended that a profile use a unique "category" within a Statement's context to refer to any Statement  
+which follows the profile.  An example profile is <a href="https://github.com/AICC/CMI-5_Spec_Current/tree/sandstone-release"> cmi5 </a>, which fits the 
+traditional single learner, single online learning use case.
+
+xAPI Communities of Practice are groups that have common goals for tracking a certain domain that can utilize xAPI.  
+A CoP typically already has a certain vocabulary used, processes, and values.  xAPI CoPs can define their own verbs,
+activity types, extensions, and other parts of statements and resources to align with their group's needs.  
+
+CoPs are highly recommended to avoid duplication of effort, as creating too many ways to solve the same problem 
+will cause fragmentation in similar domains and will hurt interoperability.  An example of a CoP for the medical 
+field is the <a href="http://groups.medbiq.org/medbiq/display/XIG/XAPI+Interest+Group+Home">Medbiquitous xAPI Interest 
+Group </a>.
 
-### 8.3 Communities of Practice
+The text below will be expanded to include all "coined" parts
```

Comment:
I don't like including notes within the document itself - i'd rather this was an issue so it doesn't get missed.